modules/fedimint-ln-client/src/lib.rs

```diff
@@ -663,14 +664,28 @@ struct GetGatewayRequest {
     force_internal: bool,
 }
 
-#[derive(thiserror::Error, Debug, Clone)]
+#[derive(thiserror::Error, Debug)]
 pub enum PayBolt11InvoiceError {
     #[error("Previous payment attempt({}) still in progress", .operation_id.fmt_full())]
     PreviousPaymentAttemptStillInProgress { operation_id: OperationId },
     #[error("No LN gateway available")]
     NoLnGatewayAvailable,
-    #[error("Funded contract already exists: {}", .contract_id)]
+    #[error("All gateways failed")]
+    AllGatewaysFailed,
```

Comment:
Can we make that error message more descriptive? Something like `No Lightning Gateway was reachable`